grasp_utils/handeye_dashboard/README.md

```diff
@@ -0,0 +1,100 @@
+# handeye_dashboard
+
+## 1.Prerequisite
+
+* System install
+  * Install [ROS2 Dashing](https://index.ros.org/doc/ros2/Installation/Dashing/Linux-Install-Debians/)
+* Install [handeye](https://github.com/RoboticsYY/handeye)
+* Install [criutils](https://github.com/RoboticsYY/criutils)
+* Install [baldor](https://github.com/RoboticsYY/baldor)
+* Install [handeye_tf_service](https://github.com/intel/ros2_grasp_library/tree/master/grasp_utils/handeye_tf_service)
+* Install [handeye_target_detection](https://github.com/intel/ros2_grasp_library/tree/master/grasp_utils/handeye_target_detection)
+
+## 2. Build and install
+
+Build with the `ros2_grasp_library` package.
```

Comment:
to be specific, just mention "colcon build --symlink-install"

Reply:
Done. Added link to the installation instructions of `ros2_grasp_library`.